pkg/consts/consts.go

```diff
@@ -37,9 +37,6 @@ const (
 	// Identity denotes identity value.
 	Identity = AppName + "-min-io"
 
-	// Namespace denotes the namespace where the app is installed
-	Namespace = Identity
```

Comment:
Can we keep `Namespace = AppName` so that it will be easier in the future for modifications.

Reply:
That's how it is https://github.com/balamurugana/directpv/blob/Rename-namespace-to-directpv/pkg/installer/consts.go#L25